src/components/document-viewer/doc-fragment-field.jsx

```diff
@@ -12,7 +15,12 @@ const DocFragmentField = ({ title, data, schema }) => {
     const Display = types[type].Display
     return <Display data={data} />
   }
-  return <div>{data}</div>
+  return (
+    <div
+      className='markdown-body'
+      dangerouslySetInnerHTML={{ __html: converter.makeHtml(data) }}
```

Comment:
Do you need to worry about injection here? Does the converter do any sort of salinization?

Reply:
The converter doesn't do any sanitisation out of the box. I'm going to create a separate issue for mitigating XSS attacks --> https://github.com/resin-io/pensieve/issues/146